<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lWhatSearch">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Searching Projects</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lHButt">
     <item>
      <widget class="QPushButton" name="bSProjects">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Search Projects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bSUsers">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Search Users</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lHID">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>ID Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lHName">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wProjects" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="lHManager">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Manager:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wUsers" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="lHRole">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Role:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lhLocation">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lHExpertise">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Expertise:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bSearch">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bSProjects</sender>
   <signal>clicked()</signal>
   <receiver>wUsers</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSProjects</sender>
   <signal>clicked()</signal>
   <receiver>wProjects</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSUsers</sender>
   <signal>clicked()</signal>
   <receiver>wProjects</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSUsers</sender>
   <signal>clicked()</signal>
   <receiver>wUsers</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
